server/src/main/java/io/druid/server/listener/resource/ListenerResource.java

```diff
@@ -98,6 +98,27 @@ public Response serviceAnnouncementPOSTAll(
     }
   }
 
+  @POST
+  @Path("/updates")
```

Comment:
This might be a good place to put the load/drop info, maybe as a return for a GET off of this or something?